2506.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @param {string[]} words
+ * @return {number}
+ */
+var similarPairs = function (words) {
+  let uniqArr = [];
+  uniqArr = words.map((word) => {
+    return [...new Set(word)].sort().join("");
+  });
+
+  let count = 0;
+  for (let i = 0; i < uniqArr.length; i++) {
+    for (let j = i + 1; j < uniqArr.length; j++) {
+      if (uniqArr[i] === uniqArr[j]) {
+        count++;
+      }
+    }
+  }
```

Comment:
```suggestion
  for (let i = 0; i < uniqArr.length - 1; i++) {
      if (uniqArr[i] === uniqArr[i + 1]) {
        count++;
      }
  }
```
You can also find the pairs without nested loops.